include/boost/beast/core/dynamic_buffer.hpp

```diff
@@ -0,0 +1,310 @@
+//
+// Copyright (c) 2020 Richard Hodges (hodges.r@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+#ifndef BOOST_BEAST_CORE_DYNAMIC_BUFFER_HPP
+#define BOOST_BEAST_CORE_DYNAMIC_BUFFER_HPP
+
+#include <boost/beast/core/detail/config.hpp>
+#include <boost/beast/core/detail/buffers_pair.hpp>
+#include <boost/beast/core/detail/dynamic_buffer_v0.hpp>
+#include <boost/asio/associated_allocator.hpp>
+#include <boost/asio/buffer.hpp>
+#include <boost/assert.hpp>
+#include <boost/type_traits/make_void.hpp>
```

Comment:
Isn't there a `make_void` in `<boost/beast/detail/config.hpp>`? I try not to include these little Boost headers, they will have to come out anyway when we do standalone beast.

Reply:
found it. At the moment, we're importing the boost make_void in the beast type_traits header, but I guess this can change.